tests/platform_tests/reboot_timing_constants.py

```diff
@@ -63,7 +63,8 @@
         "SYNCD_CREATE_SWITCH|Start": re.compile(
             r'.*syncd#syncd.*performWarmRestart: switches defined in warm restart.*'),
         "SYNCD_CREATE_SWITCH|End": re.compile(
-            r'.*syncd#syncd.*performWarmRestartSingleSwitch: Warm boot: create switch VID.*'),
+            r'.*syncd#syncd.*(performWarmRestartSingleSwitch: Warm boot: create switch VID.*|' +
+            r'brcm_sai_create_switch:\d+ SAI Timing: SDK init time \d+ seconds)'),
```

Comment:
I think SDK init time is not end of syncd create switch. There are still some actions remaining after SDK init as part of syncd create switch. Please check once again.

Reply:
You're right, there's a lot more. This log message is the latest that I can see that is coming from create switch: `Sep 18 21:33:25.452856 str-s6100-acs-2 INFO syncd#syncd: [none] SAI_API_ACL:_brcm_sai_acl_tbl_grp_mbr_migration:4685 Updated object id acl grp_mbr 0.`. However, there's still more after this point, but I don't see any reliable log message for this.

Reply:
This particular log message also won't work for DNX, looks like that might have a different code path (or not do this at all).